src/service/data-util.service.ts

```diff
@@ -103,4 +103,17 @@ export class JhiDataUtils {
     private formatAsBytes(size: number): string {
         return size.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ' ') + ' bytes';
     }
+
+    setFileData(event, entity, field: string, isImage: boolean) {
```

Comment:
Opened a PR in the main generator for that https://github.com/jhipster/generator-jhipster/pull/6206